.ruby-version

```diff
@@ -0,0 +1 @@
+3.0.2
```

Comment:
What's the reason for the bump?

Reply:
I've been having some environment issues and couldn't run `bundle install` so I changed to 3.0.2 to see if that solved my issue. It was not intentional to change it in main too. Sorry about that. We can stick to 3.0.0. and I'll try solve on my command line the problems I'm having.

Reply:
Gotcha, zero worries at all, was just curious.

Also, I have no issues with bumping to 3.0.2 on main, but we should ideally just make the change in one place (e.g. Gemfile) if possible.

Reply:
This is funny... I'm now reviewing this change to remember when/why I did it and I see the Gemfile and Gemfile.lock are untouched... but there is a file `.ruby-version` where this `3.0.2` Ruby version is specified that I don't remember I've created...

Reply:
I've manually deleted the `.ruby-version` file from the directory